packages/editor/src/components/entities-saved-states/index.js

```diff
@@ -36,56 +27,31 @@ function identity( values ) {
 	return values;
 }
 
-function isPreviewingTheme() {
+export default function EntitiesSavedStates( { close } ) {
+	const isDirtyProps = useIsDirty();
 	return (
-		window?.__experimentalEnableThemePreviews &&
-		getQueryArg( window.location.href, 'theme_preview' ) !== undefined
+		<EntitiesSavedStatesExtensible
+			{ ...{
+				...isDirtyProps,
+				close,
+			} }
+		/>
```

Comment:
```suggestion
		<EntitiesSavedStatesExtensible
		        close,
			{ ...isDirtyProps }
		/>
```

Will this not work?

Reply:
I addressed it with your suggestion! Thanks!
https://github.com/WordPress/gutenberg/pull/50863/commits/032a8b65a3fd9cf6769f9a0052d829820392b234